editor/engine-features/render-config.json

```diff
@@ -224,7 +224,7 @@
             "category": "animation"
         },
         "custom-pipeline": {
-            "default": ["custom-pipeline"],
+            "default": [],
```

Comment:
I want to disable custom-pipeline by default. Is this change correct?

Reply:
It's OK.